template_config.yml

```diff
@@ -41,11 +41,12 @@ pulp_settings:
   allowed_import_paths:
   - /tmp
 pulpcore_branch: '3.12'
-pulpcore_pip_version_specifier: '~=3.12.0'
+pulpcore_pip_version_specifier: ~=3.12.0
```

Comment:
don't we want to use 3.13?

Reply:
+1 for both `pulpcore_branch:` and `pulpcore_pip_version_specifier:`

Reply:
The rpm/3.12 release requirements.txt specifies >=core-3.12. I don't know how the failing-test in question ever passed, because it's in this branch, and the corresponding core-commit is *not* in 3.12 it's in 3.13. Changing the template will get CI working, but is it the right thing to do?